docker/docker-compose.2204.510.yaml

```diff
@@ -3,28 +3,27 @@ version: "3"
 services:
```

Comment:
In addition to this, do you want to update the [Swift 5.9 pipeline](https://github.com/apple/swift-nio/blob/main/docker/docker-compose.2204.59.yaml) to use a release rather than nightly image?

Reply:
Yes, please